src/content/learn/responding-to-events.md

```diff
@@ -220,22 +220,22 @@ button { margin-right: 10px; }
 
 </Sandpack>
 
-Here, the `Toolbar` component renders a `PlayButton` and an `UploadButton`:
+在這裡，`Toolbar` component render 了一個 `PlayButton` 和一個 `UploadButton`：
 
-- `PlayButton` passes `handlePlayClick` as the `onClick` prop to the `Button` inside.
-- `UploadButton` passes `() => alert('Uploading!')` as the `onClick` prop to the `Button` inside.
+- `PlayButton` 將 `handlePlayClick` 作為 `onClick` prop 傳遞給內部的 `Button`。
+- `UploadButton` 將 `() => alert('Uploading!')` 作為 `onClick` prop 傳遞給內部的 `Button`。
 
-Finally, your `Button` component accepts a prop called `onClick`. It passes that prop directly to the built-in browser `<button>` with `onClick={onClick}`. This tells React to call the passed function on click.
+最後，你的 `Button` component 接受一個名為 `onClick` 的 prop，它將該 prop 透過 `onClick={onClick}` 直接傳遞給瀏覽器內建的 `<button>`。這告訴 React 在點擊時呼叫被傳遞的函式。
 
-If you use a [design system](https://uxdesign.cc/everything-you-need-to-know-about-design-systems-54b109851969), it's common for components like buttons to contain styling but not specify behavior. Instead, components like `PlayButton` and `UploadButton` will pass event handlers down.
+如果你使用 [設計系統](https://uxdesign.cc/everything-you-need-to-know-about-design-systems-54b109851969)，像按鈕這樣的 component 只包含樣式但不指定行為是很常見的。相反地，像 `PlayButton` 和 `UploadButton` 這樣的 component 會向下傳遞 event handler。
```

Comment:
What do you think keep the `design system` here?

```suggestion
如果你使用[設計系統](https://uxdesign.cc/everything-you-need-to-know-about-design-systems-54b109851969)，像按鈕這樣的 component 只包含樣式但不指定行為是很常見的。相反地，像 `PlayButton` 和 `UploadButton` 這樣的 component 會向下傳遞 event handler。
```

Reply:
I agree.